src/main.c

```diff
@@ -238,6 +239,7 @@ CF_ABS("auth_query", CF_STR, cf_auth_query, 0, "SELECT usename, passwd FROM pg_s
 CF_ABS("auth_type", CF_LOOKUP(auth_type_map), cf_auth_type, 0, "md5"),
 CF_ABS("auth_user", CF_STR, cf_auth_user, 0, NULL),
 CF_ABS("autodb_idle_timeout", CF_TIME_USEC, cf_autodb_idle_timeout, 0, "3600"),
+CF_ABS("cache_vars", CF_STR, cf_cache_vars, 0, NULL),
```

Comment:
All tests in CI are failing because the option is not in the docs.
```
etc/optscan.sh
cache_vars is missing in doc/config.md
cache_vars is missing in etc/pgbouncer.ini
```

Lets add it. Also lets name the option `extra_tracked_startup_parameters`.